src/components/theme-switcher/user-dropdown-switcher/user-dropdown-switcher.module.css

```
@@ -0,0 +1,15 @@
+.label {
+	composes: hds-typography-body-200 from global;
+	composes: hds-font-weight-medium from global;
+}
+
+.listItem {
+	display: flex;
+	border-radius: 5px;
+	color: var(--token-color-foreground-primary);
+	width: 100%;
+	border: none;
+	padding: 8px 16px 8px 16px;
+	gap: 12px;
+	align-items: center;
+}
```

🎨 `border-radius`, `width`, and `border` aren't doing anything for us on this class, so they can be removed.

```suggestion
.listItem {
	display: flex;
	color: var(--token-color-foreground-primary);
	padding: 8px 16px 8px 16px;
	gap: 12px;
	align-items: center;
}
```